src/app/shared/components/project-modal/project-modal.component.ts

```diff
@@ -60,6 +62,18 @@ export class ProjectModalComponent implements OnInit {
     }
 
     /*OCR Mechanism*/
+    async ocrFunc(fileLink: string) {
+        const vision = require('@google-cloud/vision');
+
+        // Creates a client
+        const client = new vision.ImageAnnotatorClient();
+
+        // Performs text detection on the gcs file
+        const [result] = await client.textDetection(`${fileLink}`);
+        const detections = result.textAnnotations;
+        console.log('Text:');
+        detections.forEach((text: any) => console.log(text));
```

Comment:
We're gonna want to build a string that we can save back to the Firebase Real-Time Database.